app/controllers/api/v1/widget/messages_controller.rb

```diff
@@ -11,6 +11,10 @@ def index
   def create
     @message = conversation.messages.new(message_params)
     @message.save!
+    if params[:message][:attachment].present?
+      @message.attachment = Attachment.new(account_id: @message.account_id)
```

Comment:
Can we make this as `@message.build_attachment(account_id: @message.account_id)` ?

Reply:
Since this is on the widget side, I'll fix this with another PR to make uploads on widget side